opentracing-spring-cloud-starter/src/test/java/io/opentracing/contrib/spring/cloud/TestNoDeps.java

```diff
@@ -0,0 +1,30 @@
+package io.opentracing.contrib.spring.cloud;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author Pavol Loffay
+ */
+public class TestNoDeps {
+
+    @Test
+    public void testNoFeign() {
+        try {
+            this.getClass().getClassLoader().loadClass("feign.Client");
+            this.getClass().getClassLoader().loadClass("feign.hystrix.HystrixFeign");
+            Assert.fail();
+        } catch (ClassNotFoundException e) {
+        }
+    }
+
+    @Test
+    public void testNoJMS() {
+        try {
+            this.getClass().getClassLoader().loadClass("javax.jms.Message");
+            this.getClass().getClassLoader().loadClass("org.springframework.jms.core.JmsTemplate");
```

Comment:
Should these be in their own try/catch blocks to test individually - as they will be in different maven dependencies - either of which could be included separately. Same applies to the feign test above.